test/functional/fixtures/nvim/after/filen.lua

```diff
@@ -0,0 +1 @@
+table.insert(_G.fileorder, "ordinary after")
```

Comment:
name is a bit generic, maybe it's worth helping  `grep` attempts:
```suggestion
table.insert(_G.test_loadorder, "ordinary after")
```